src/popup/utils/constants.ts

```diff
@@ -68,6 +72,69 @@ export const TX_FUNCTIONS = {
   claim: 'claim',
 } as const;
 
+export const TX_FUNCTION_TYPE_MULTISIG = {
+  propose: 'propose',
+  confirm: 'confirm',
+  refuse: 'refuse',
+  revoke: 'revoke',
+} as const;
+
+export const TX_FUNCTION_TYPE_DEX: Record<DexFunctionType, TxFunctionRaw[]> = {
+  pool: [
+    'remove_liquidity', 'remove_liquidity_ae', 'add_liquidity', 'add_liquidity_ae',
+  ],
+  removeLiquidity: [
+    'remove_liquidity', 'remove_liquidity_ae',
+  ],
+  addLiquidity: [
+    'add_liquidity', 'add_liquidity_ae',
+  ],
+  swap: [
+    'deposit', 'withdraw', 'swap_exact_tokens_for_tokens', 'swap_tokens_for_exact_tokens',
+    'swap_exact_ae_for_tokens', 'swap_tokens_for_exact_ae', 'swap_exact_tokens_for_ae',
+    'swap_ae_for_exact_tokens',
+  ],
+  allowance: [
+    'transfer_allowance', 'change_allowance', 'create_allowance',
+  ],
+  maxSpent: [
+    'swap_tokens_for_exact_tokens', 'swap_tokens_for_exact_ae', 'swap_ae_for_exact_tokens',
+  ],
+  minReceived: [
+    'swap_exact_tokens_for_tokens', 'swap_exact_ae_for_tokens', 'swap_exact_tokens_for_ae',
+  ],
+};
+
+/**
+ * ITx.tag
+ */
+export const TX_TAGS_SUPPORTED = [
```

Comment:
```suggestion
export const SUPPORTED_TX_TAGS = [
```

Reply:
Also here I think starting from more general part and finish with specifics is a more well laid out approach.